extensions/parquet/table/build.gradle

```diff
@@ -48,6 +51,9 @@ dependencies {
             project(path: ':test-configs')
     Classpaths.inheritSlf4j(project, 'slf4j-simple', 'testRuntimeOnly')
 
+    implementation project(':plugin-trackedseekablechannel')
```

Comment:
Please share your suggestions on the names of these plugins and where exactly to place them

Reply:
Commented in the settings.gradle.